nginx/amiable.sh

```diff
@@ -5,7 +5,15 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 nginxHome=`nginx -V 2>&1 | grep "configure arguments:" | sed 's/[^*]*conf-path=\([^ ]*\)\/nginx\.conf.*/\1/g'`
 
-sudo ln -fs $DIR/amiable.conf $nginxHome/sites-enabled/amiable.conf
+if [ "$(uname)" == "Darwin" ]; then
+    # Mac OS X platform
+    confDir=servers
+elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+    #  GNU/Linux platform
+    confDir=sites-enabled
+fi
+
```

Comment:
I think you can simply do `SYSTEM=$(uname -s)` and compare it to `Linux` and `Darwin`. That should work in both case